esteid/templates/esteid/test-new.html

```diff
@@ -0,0 +1,445 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>ESTEID Test Page</title>
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+</head>
+
+<body>
+<div class="container">
+    <h1>Esteid Test page</h1>
+    <p>
+        This is a test page for digital signing of files via python.
+    </p>
+
+    <ul>
+        <li>
+            Mobile ID signing is done with Demo Mobile ID service, you can only use the
+            <a href="https://github.com/SK-EID/MID/wiki/Test-number-for-automated-testing-in-DEMO" target="_blank">demo
+                test numbers</a>.
+            The signatures will not be valid unless the test certificates are used to validate them.
+        </li>
+        <li>
+            ID Card signing can be done with real ID cards, but such a signature can not be validated.
```

Comment:
It is there in the readme. I guess a link to it would do, this view is there for testing not for documentation, I know assumption is mother of all evils but I still assume people won't end up on this page by accident.